requirements.d/development.txt

```diff
@@ -23,3 +23,6 @@ pytest-reverse
 eventlet>0.30.2
 dnspython>=2.0.0, <2.3.0
 gsl==2.7.0
+Flask-WTF
```

Comment:
the new ones need to be moved to localbuild meta.yaml, both places will create a new testing container